content/news/016/index.md

```diff
@@ -30,7 +30,6 @@ Feel free to send PRs about your own projects!
 
 Table of contents:
 
-- [Rust GameDev Podcast](#rust-gamedev-podcast)
```

Comment:
This heading does exist, but it should be `[Rust GameDev Podcast #3](#rust-gamedev-podcast-3)`.